pkg/loki-curator/config/config.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// CuratorConfig holds the curator's configurations
+type CuratorConfig struct {
+	LogLevel        string        `yaml:"LogLevel,omitempty"`
+	DiskPath        string        `yaml:"DiskPath,omitempty"`
+	TriggerInterval int           `yaml:"TriggerInterval,omitempty"`
+	InodeConfig     InodeConfig   `yaml:"InodeConfig,omitempty"`
+	StorageConfig   StorageConfig `yaml:"StorageConfig,omitempty"`
+}
+
+// InodeConfig holds the curator's inode configuration
+
+type InodeConfig struct {
+	MinFreePercentages             int `yaml:"MinFreePercentages,omitempty"`
+	TargetFreePercentages          int `yaml:"TargetFreePercentages,omitempty"`
+	PageSizeForDeletionPercentages int `yaml:"PageSizeForDeletionPercentages,omitempty"`
+}
+
+// StorageConfig holds the curator's storage configuration
+type StorageConfig struct {
+	MinFreePercentages             int `yaml:"MinFreePercentages,omitempty"`
+	TargetFreePercentages          int `yaml:"TargetFreePercentages,omitempty"`
+	PageSizeForDeletionPercentages int `yaml:"PageSizeForDeletionPercentages,omitempty"`
+}
+
```

Comment:
```suggestion
```
This is the same as InodeConfig. Use some common name for this config structure.